notebooks/samples/tensorflow/prediction_auto-scaling/ai_platform_prediction_auto-scaling.ipynb

```diff
@@ -0,0 +1,422 @@
+{
```

Comment:
<p>For the second sentence ("The <code>gcloud</code> beta CLI"...), it's not immediately clear that you're referring to <code>gcloud beta ai-platform versions create</code></p><p> </p><p>So I'd rewrite it as:</p><p>"Alternatively, you can use [<code>gcloud beta ai-platform versions create</code>](https://cloud.google.com/sdk/gcloud/reference/beta/ai-platform/versions/create#--metric-targets) to specify the parameters directly without <code>config.yaml</code>:"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/231/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
I've clarified the wording. Thank you.